games-misc/cowsay/cowsay-3.04.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="configurable talking ASCII cow (and other characters)"
```

Comment:
Let's capitalise this while we're at it.

Reply:
wow, that's eye for detail :)
fixed